### PR TITLE
[14.0][ADD] shopinvader_sale_amount_by_group

### PR DESCRIPTION
**Pull request by ivantodorovich**
This module exposes to shopinvader the `amount_by_group` field on sale orders.
This field contains the tax amount grouped by tax group.

### CODE REVIEW

File: shopinvader_sale_amount_by_group/tests/test_cart.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        self.tax_10 = self.env.ref("shopinvader_sale_amount_by_group.tax_10")
+        self.tax_20 = self.env.ref("shopinvader_sale_amount_by_group.tax_20")
+        self.cart.order_line[0].tax_id = [(6, 0, self.tax_10.ids)]
+        self.cart.order_line[1].tax_id = [(6, 0, self.tax_20.ids)]
+
+    def test_cart(self):
+        data = self.service.dispatch("update", params=dict())["data"]
+        groups = [g["name"] for g in data["amount"]["amount_by_group"]]
+        self.assertEqual(
+            groups, (self.tax_10 | self.tax_20).mapped("tax_group_id.name")
```
Comment (jcoux) [inline shopinvader_sale_amount_by_group/tests/test_cart.py]: I would have also base and amount in unit test.

**Review by jcoux — APPROVED** [review]
One comment not blocking :+1:

File: shopinvader_sale_amount_by_group/tests/test_cart.py (hunk 1 above)
Comment (simahawk) [reply shopinvader_sale_amount_by_group/tests/test_cart.py]: agree

**Review by simahawk — APPROVED** [review]

**Pull request merged**